### DISCUSSION TITLE
Can an Airgap install be reverted to non-Airgap once the system is live?

### DISCUSSION THREAD

**Discussion by anuzis** [Security-Onion-Solutions/securityonion#3813] [Unsupported Versions]
(First time installing Security Onion; residential use on an Intel NUC with 16GB ram, 500GB SSD, quad core; the NUC has a wireless adapter, but there didn't appear to be an option to enable it for the install mode.)

I'm conducting the install in a room without ethernet access, but once the install finishes I'll move the device where it will have internet access (management interface with a static IP defined for the local network; monitoring interface listening in a gigabit network tap).

From the docs ( https://docs.securityonion.net/en/2.3/airgap.html ), it sounds like if you choose an Airgap install, SecurityOnion assumes it will never have internet access and will expect all IDS rules updates/etc to be delivered by hand via USB.  I'd like SecurityOnion to take advantage of the live internet access it will have once it's plugged in.

Is there a way I can adjust the SO configuration post-install so it can take advantage of the live internet?

Alternatively, I could redo the install with the ethernet connection in place, but that'd require moving some desks or laying the monitor, keyboard, & mouse on the floor where the device will be installed. Definitely an option if that's the recommendation, just thought it may be possible to let SO know it's only temporarily airgapped during the install process itself.

Appreciate any guidance folks can provide.

**Comment by anuzis** [comment]
Update: No reply needed. I was excited to get everything rolling and just moved the monitor/keyboard/mouse onto the floor in the server area, redid a fresh install via ethernet and everything is up and rolling. Excited to try Security Onion!  Thanks to everyone who helps make this software possible.

**Comment by HelenH321** [comment]
Please could we have an answer to this one after all??  I have installed as airgapped on a system which is soon to go live and have now been provided with internet access.  I would very much like to revert to internet access without having to reinstall due to time constraints and other configuration which has been applied.  Can anyone help?
Thanks.

**Reply by palichx** [reply]
Hi,
i think you need to change  "airgap: True" to  "airgap: False" in pillar file:
/opt/so/saltstack/local/pillar/global.sls.

**Reply by HelenH321** [reply]
Thanks palichx, do you know if there are any wider implications across the system?  I will give it a try and report back :)

**Reply by HelenH321** [reply]
Struggling with this - understandably, the networking isn't set up to allow updating.  Going to reinstall after all - I'll get it right first time next time.  Thanks for your help, palichx!
